lib/dolly/document.rb

```diff
@@ -128,7 +128,8 @@ def write_property name, value
     end
 
     def read_property name
-      self.properties[name].value
+      return self.properties[name].value unless doc[name.to_s]
```

Comment:
actually I think this is more a problem with the `Dolly::Document` class, the issue being that I don't want to have to call `doc[some_key]` every time I want to access a property correctly.  I would must rather just call the read method.  I don't think this can be hidden in the property class because the issue is really with the `doc` attribute and how to read the properties from there.